manifests/c/COAS/Donkey/20.6.25.0/COAS.Donkey.locale.en-US.yaml

```
@@ -0,0 +1,9 @@
+PackageIdentifier: COAS.Donkey
+PackageVersion: 20.6.25.0
+PackageLocale: en-US
+Publisher: COAS, your partner in computer aided services
```

```suggestion
Publisher: COAS software systems
```
This must match Control Panel.